internal/ethapi/api.go

```diff
@@ -778,27 +778,35 @@ func (s *PublicBlockChainAPI) EstimateGas(ctx context.Context, args CallArgs) (h
 	cap = hi
 
 	// Create a helper to check if a gas allowance results in an executable transaction
-	executable := func(gas uint64) bool {
+	executable := func(gas uint64) (bool, error) {
```

Comment:
Just return the error, because you can just check `err == nil` instead of having this bool.

Reply:
This follows the pattern of returning (bool, error) throughout the celo-blockchain codebase- any reason not to be consistent?